src/Orc.Automation/Helpers/FunctionNameFormatHelper.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Orc.Automation;
+
+using System.Linq;
+using System.Text;
+using Catel;
+
+public static class FunctionNameFormatHelper
+{
+    public static string FormatMethodName(string methodName, params object[] args)
+    {
+        var stepNameBuilder = new StringBuilder(methodName.SplitCamelCase());
+
+        if (args.Length > 0)
+        {
+            stepNameBuilder.Append(" (");
+        }
+
+        var argIndices = Enumerable.Range(0, args.Length)
+            .Select(index => $"'{{{index}}}'");
+
+        stepNameBuilder.AppendJoin(',', argIndices);
```

Comment:
AppendJoin(", "), or don't you like the spaces between arguments?

Reply:
oh...finally understand...",_space_" - didn't noticed first time...

Reply:
maybe, yes it would be better